Task14

andreyv24 opened the pull request.

andreyv24 committed:
.

andreyv24 committed:
.

andreyv24 committed:
.

andreyv24 committed:
.

andreyv24 committed:
.

andreyv24 committed:
.

andreyv24 committed:
.

andreyv24 committed:
.

andreyv24 committed:
Merge branch 'KFalcon2022:master' into master

andreyv24 committed:
Merge branch 'KFalcon2022:master' into task5

andreyv24 committed:
.

andreyv24 committed:
Merge remote-tracking branch 'origin/task5' into task5

# Conflicts:
#	src/main/java/com/walking/intensive/chapter1/task5/Task5.java

andreyv24 committed:
Merge branch 'KFalcon2022:master' into master

andreyv24 committed:
.

andreyv24 committed:
.

Zawadskiy commented on src/main/java/com/walking/intensive/chapter1/task5/Task5.java:
```diff
@@ -11,6 +11,9 @@
 public class Task5 {
```
Task5? Убрать из pr.

andreyv24 replied:
Удалил.

Zawadskiy commented on src/test/java/com/walking/intensive/chapter1/task5/Task5Test.java:
```diff
@@ -29,6 +29,7 @@ void getAreaByHeronTest() {
     void getHeightsTest() {
```
и єто тоже убрать

andreyv24 replied:
Удалил.

Zawadskiy commented on src/main/java/com/walking/intensive/chapter3/task14/Task14.java:
```diff
@@ -43,11 +45,49 @@
+
     static int[] getObjectCounts(int[][] objectLocations, int[][] radars) {
-        // Ваш код
-        return new int[0];
```
де пустііе строки между логическими блоками?)

andreyv24 replied:
Добавил.

Zawadskiy commented on src/main/java/com/walking/intensive/chapter3/task14/Task14.java:
```diff
@@ -43,11 +45,49 @@
-        // Ваш код
-        return new int[0];
+        int[] objectCounts = new int[radars.length];
+        for (int j = 0; j < radars.length; j++) {
```
Почему j?) Куда i пропало?)

andreyv24 replied:
Сложный вопрос. )))) Заменил на i.

Zawadskiy commented on src/main/java/com/walking/intensive/chapter3/task14/Task14.java:
```diff
@@ -43,11 +45,49 @@
+                if (objectLocation.length != 2) {
+                    return new int[0];
+                }
+                if ((Math.pow(objectLocation[0] - radars[j][0], 2) + Math.pow(objectLocation[1] - radars[j][1], 2)) <= Math.pow(radars[j][2], 2)) {
```
> Task :checkstyleMain
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter3/task14/Task14.java:85: Line is longer than 120 characters (found 147). [LineLength]

andreyv24 replied:
Сделал перенос на другую строку.

andreyv24 committed:
Delete src/main/java/com/walking/intensive/chapter1/task5/Task5.java

andreyv24 committed:
Delete src/test/java/com/walking/intensive/chapter1/task5/Task5Test.java

andreyv24 committed:
- Заменил j на i.
- Разделил пустыми строками логические блоки.
- Сделал перенос длинной строки более 120 символов.

andreyv24 committed:
Merge remote-tracking branch 'origin/task14' into task14

# Conflicts:
#	src/main/java/com/walking/intensive/chapter1/task5/Task5.java
